oracle/datadog_checks/oracle/oracle.py

```diff
@@ -230,6 +239,23 @@ def _get_sys_metrics(self, con, tags):
                 if metric_name in self.SYS_METRICS:
                     self.gauge(self.SYS_METRICS[metric_name], metric_value, tags=tags)
 
+    def _get_process_metrics(self, con, tags):
+        if tags is None:
+            tags = []
+
+        query = "SELECT PROGRAM, {} FROM GV$PROCESS".format(','.join(self.PROCESS_METRICS.keys()))
+        with closing(con.cursor()) as cur:
+            cur.execute(query)
+            for row in cur.fetchall():
+
+                # Oracle program name
+                program_tag = ['program:{}'.format(row[0])]
+
+                # Get the metrics
+                for i, metric_name in enumerate(self.PROCESS_METRICS.values(), 1):
```

Comment:
what will this output look like? is this relying on ordering? Does the db always keep that well ordered?

Reply:
This is related to the previous comment :)